lua/ibl/utils.lua

```diff
@@ -231,4 +246,133 @@ M.tbl_get_index = function(list, i)
     return list[((i - 1) % #list) + 1]
 end
 
+M.highlight_from_extmark = function(bufnr, config, start_row, start_col, end_row, end_col, fallback_index)
+    local highlight = config.scope.highlight
```

Comment:
```suggestion
M.highlight_from_extmark = function(bufnr, config, start_row, start_col, end_row, end_col, fallback_index, highlight)
```
`current_indent_highlight_from_extmark` also uses this, but it might have highlights that differ from the scope highlights.